3scale/3scale-operator
# Add labels to 3scale capabilities

# What
Add owner reference label to products cr to help with cascading deletion

# Verification

1.  Install RHOAM locally from this branch: https://github.com/integr8ly/integreatly-operator/pull/2409 but replace the manifests/3scale/0.9.0 values for container image and image from quay.io/austincunningham/3scale-operator:cluster-scoped to quay.io/pstefans/3scale-operator:v0.101.0
2. Wait for the installation to complete
3. Create product via CR
4. Confirm product CR has a new `ownerRef` label.

Note: Guide for creating products through CRs https://docs.google.com/document/d/16CTVrFtsDR27_ZS9-7_xS9JKwteSmi7I_IFV7DShuXI/edit
